Documentation/git-revert.txt

```diff
@@ -101,6 +102,10 @@ effect to your index in a row.
 	Pass the merge strategy-specific option through to the
 	merge strategy.  See linkgit:git-merge[1] for details.
 
+--skip::
+	Skip the current commit and continue reverting the rest
+	of the sequence.
```

Comment:
Should these docs be added to `sequencer.txt` instead?  Currently that is included only in `git-revert.txt` and `git-cherry-pick.txt`, and all commands that fully support the sequencer subcommands should probably implement `--skip` anyway.